static/src/stylesheets/inline/story-package-garnett.scss

```diff
@@ -372,6 +371,78 @@ $fc-item-gutter: $gs-gutter / 4;
                 display: none;
             }
         }
+
+        // Invert the live update colours for these cards
+        &.fc-item--full-media-100-tablet, &.fc-item--three-quarters-tall-tablet {
```

Comment:
How much is repeated here? You could pull some of the styles into a mixin taking the colours as params and then `@include` it? Not sure if that's helpful here!

Reply:
Good question, there is some amount of duplication going on. I'll look into that. If it looks like being non-trivial I might put it off for a future PR to get this out sooner.

Reply:
Yup!